app/components/Dashboard/Cards/ResaleValue.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import GraphCard from "../../ui/GraphCard";
+import ResaleValueWrapper from "../../graphs/ResaleValueWrapper";
+import { Drawer } from "../../ui/Drawer";
+import { Household } from "@/app/models/Household";
+import TenureSelector from "../../ui/TenureSelector";
+
+interface DashboardProps {
+  data: Household;
+}
+
+export const ResaleValue: React.FC<DashboardProps> = ({ data }) => {
+  const [selectedTenure, setSelectedTenure] = useState<'landPurchase' | 'landRent'>('landPurchase');
+
+  return (
+    <GraphCard
+      title="How much could I sell it for?"
+      subtitle="Estimated sale price at any time"
+    >
+      <div className="flex flex-col h-full w-3/4 justify-between">
+        <div className="flex gap-2 mb-4">
+          <TenureSelector 
+            isSelected={selectedTenure === 'landPurchase'}
```

Comment:
nit: We favour `"` over `'` for strings (repeated a few times in this component)